### ISSUE TITLE
Remove the apt-get part in quick installation. It's confusing!

### ISSUE THREAD

**Issue by joezhouchenye** [powerline/fonts#361]
If you run `sudo apt-get install ...` l in ubuntu, only fonts named `Powerline *` will be installed, instead of  all the fonts listed.

**Comment by TZECHIN6** [comment]
Agree... its quite confusing, after apt-get install and search fonts in the preference tag... No fonts can be found.

**Comment by stytchiz** [comment]
+1. And there should be follow-up instructions as described in https://github.com/powerline/fonts/issues/281#issuecomment-1088154652 to make the fonts discoverable.
